modules/claims_api/lib/claims_api/error/soap_error_handler.rb

```diff
@@ -41,6 +41,8 @@ def get_exception
         )
       elsif record_not_found?
         raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Record not found.')
+      elsif veteran_not_found?
+        { status: '404', detail: 'Veteran ID not found' }
```

Comment:
Return custom object to be handled in our controller for this specific scenario so we don't return 404 on ALL empty responses from BGS.